dms_version/models/dms_file.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017-2020 MuK IT GmbH
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class DmsFile(models.Model):
+    _inherit = "dms.file"
+
+    version_ids = fields.One2many(
+        comodel_name="dms.version",
+        inverse_name="file_id",
+        string="Versions",
+        readonly=True,
+    )
+    count_versions = fields.Integer(
+        compute="_compute_count_versions", string="Count Versions"
+    )
```

Comment:
This is English! 😃 

```suggestion
    versions_count = fields.Integer(
        compute="_compute_versions_count",
        store=True,
    )
```

☝🏼 Besides I added `store=True` because I can see that the result will always be the same, and you're using the field in multirecord views (kanban, tree), so storing it will improve performance.